### DISCUSSION TITLE
data gone in Heroku [Jun 22]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2507] [Discord Threads]
> 🙋‍♂️ **mardek01**

**Comment by o1lab** [comment]
> 🙋‍♂️ **wkw**  
#### any detailed error messages shown in backend?  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/988727710752841748)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
